addons/hw_drivers/iot_handlers/drivers/L10nKeEDISerialDriver.py

```diff
@@ -188,16 +204,16 @@ def abort_post(self):
         open otherwise, blocking further invoices being sent.
         """
         self.message_number += 1
-        abort = struct.pack('BBB', 37, self.message_number + 32, 0x39)
+        abort = struct.pack('BBB', 35, self.message_number + 32, 0x39)
```

Comment:
why?

Reply:
this is the `<len>` byte
It was wrong to begin with, the actual value is 35 (the length of the message is `3`, and the offset is `0x20 (=32)`, so `32 + 3 = 35`)